compiler/src/dmd/expressionsem.d

```diff
@@ -5488,6 +5488,14 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             }
         }
 
+        // `super.fun()` with fun being abstract and unimplemented
+        auto supDotFun = exp.e1.isDotVarExp();
+        if (supDotFun && supDotFun.e1.isSuperExp() && exp.f && exp.f.isAbstract() && !exp.f.fbody)
+        {
+            exp.error("call to unimplemented abstract function `%s`", exp.f.toFullSignature());
+            exp.errorSupplemental("declared here: %s", exp.f.loc.toChars());
```

Comment:
It's better to put the supplemental error at `f.loc` so you get there when you ctrl+click the line in VSCode (and other editors too probably)

Reply:
that wont be nicely aligned anymore then

Reply:
I can patch the loc actually

Reply:
If https://github.com/dlang/dmd/pull/15646 gets through, both need to be converted to free functions

Reply:
yes I've seen that PR earlier. But for this problem I'm no convinced. Patching the loc allows to drop the "Error:" part of the message but IRL the two messages will still be unaligned (with two different modules)

Reply:
I don't understand what you mean. Alignment of `errorSupplemental` with the preceding `error` is a general problem, not specific to this particular error. Also, setting `exp.loc = x` and calling `exp.errorSupplemental(...)` is equivalent to `.errorSupplemental(x, ...)`, so I don't see how it relates to alignment.

Reply:
Allright. I'll use `.errorSupplemental` too.

Reply:
https://github.com/dlang/dmd/pull/15646 got merged, so now you need to use `.error` as well.